### PR TITLE
Update instructions and entry format.

### PR DESCRIPTION
**Pull request by hargup**

### CODE REVIEW

**Comment by hargup** [comment]
@routeaccess ping.

**Comment by hargup** [comment]
@abinashmeher999 are you ok with the new proposed changes?

File: README.md [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+
+  You should also through the [curated list of git
+resource](https://help.github.com/articles/good-resources-for-learning-git-and-github/) provided by github to find other resouces to fit your need.
+
+- **Fork the repo, create a new branch, make changes and send a pull request**
+  - [This article](https://help.github.com/articles/using-pull-requests/) should give you a good idea of what pull requests are how to use them.
+
+
+Your entry should maintain the alphabetic order and it should be in the format
+`* your name`. This file is written the [Markdown format](https://guides.github.com/features/mastering-markdown/)
```
Comment (routeaccess) [inline README.md]: This file is writtern `in` the Markdown format

**Comment by routeaccess** [comment]
Please reword detail of commit 48b12c7c7e44dc25b297279b78947b26d6b2516b - requires grammar check.

**Comment by abinashmeher999** [comment]
@hargup Wish you had done it sooner :stuck_out_tongue_winking_eye:

**Comment by hargup** [comment]
> @hargup Wish you had done it sooner

True, should have done this before we got this amount of traffic.

**Comment by hargup** [comment]
@routeaccess @abinashmeher999 I have fixed the error, have a look.

**Comment by narayanaditya95** [comment]
Looks good to me.

**Comment by routeaccess** [comment]
Yes, I find this solution much more appealing. 
This should work fine :
```
- <Name> <username_linked_to_github_profile>
```
- Nevin Valsaraj [routeaccess](http://github.com/routeaccess)

**Comment by hargup** [comment]
> Yes, I find this solution much more appealing. 
> This should work fine :
> ```
> - <Name> <username_linked_to_github_profile>
> ```

:+1: Can you open up a PR for the proposed changes?

**Comment by routeaccess** [comment]
Opened PR #26

**Pull request merged**